package.json

```diff
@@ -4,26 +4,24 @@
   "engines": {
     "node": "9.11.1"
   },
-  "dependencies": {},
+  "dependencies": {
+    "View": "git+ssh://git@github.com/FutureDaysSoftware/View.git#bundling"
```

Comment:
Why can't we use #master here?

Also, we should start publishing our packages so we don't have to put in the git branch:

https://docs.npmjs.com/misc/scope

Reply:
@futuredays/view

Reply:
#master on the View repo is still bundling with browserify, which I found out is not the way to go (can't require a browserify bundle). So I was just using the version of View that has the proper src, dist organization in order to test it with Toast. It works. Or at least there are no issues with babel compiling Toast.

As for publishing, do we want private or public scoped packages? Apparently you have to pay to have private packages in the npm registry, so that might not be worth it. I don't think we really give a shit if some random person decides to use one of our packages.

Reply:
Right, the View repo should include a babel transformed dist.

Public scoped.  It would be neat if other people used these things.

Reply:
There's a PR open for the View repo to merge the src, dist change into #master, if you didn't realize.